app/views/users/new.html.erb

```diff
@@ -61,13 +73,15 @@
     <%= content_tag :span, @new_user.errors[:faculty].first, class: 'form-error' %>
     <%= f.select :faculty, ["N/A","Arts","Education","Engineering","Law","Medicine","Science"," Social Sciences","Telfer school of Management"] , prompt: "Choose..." %><br><br>
 
+    <% programs = File.read("#{Rails.root}/public/programs.txt").lines %>
```

Comment:
You could move this into it's own class and have it loaded in an initializer `in config/initializers`. Something like

```
class ProgramList

def self.fetch_all
 @@programs ||= File.read(filename).lines
end
end
```